Makefile

```diff
@@ -0,0 +1,84 @@
+.DEFAULT_GOAL:=help
+
+BINARY?=opcap
+IMAGE_BUILDER?=podman
+IMAGE_REPO?=quay.io/opdev
+VERSION=$(shell git rev-parse HEAD)
+RELEASE_TAG ?= "0.0.0"
+
+PLATFORMS=linux
+ARCHITECTURES=amd64 arm64 ppc64le s390x
+
+.PHONY: build
+build:
+	go build -o $(BINARY) -ldflags "-X github.com/opdev/opcap/version.commit=$(VERSION) -X github.com/opdev/opcap/version.version=$(RELEASE_TAG)" main.go
```

Comment:
This target is dropping opcap on the root of the project. Can we move it to bin/opcap that is already on gitignore?

Reply:
done

Reply:
Thanks!